components/serializer.rst

```diff
@@ -605,23 +605,7 @@ There are several types of normalizers available:
 Encoders
```

Comment:
Considering that we show all the information about `Normalizers` in this article ... I think the change should be the opposite: delete the `serializer/encoders.rst` article entirely and move its contents to this `Encoders` section.

Reply:
I will do